stable/kubexray/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubexray
-version: 0.4.0
+version: 0.4.1
 appVersion: 0.1.5
 home: https://github.com/jfrog/kubexray
 description: KubeXray Helm chart
```

Comment:
Add `DEPRECATED` to the beginning of the description

Reply:
done